spec/features/ncr/work_orders/create_spec.rb

```diff
@@ -112,6 +112,17 @@
       expect(page).to_not have_content("Approving official email can't be blank")
     end
 
+    scenario "inactive users do not appear as potential approvers", :js do
```

Comment:
Doesn't hurt to have more, but these tests are pretty much duplicates in different locations. I think it makes sense to have one with js!